addons/event/tests/test_event_internals.py

```diff
@@ -56,6 +56,54 @@ def setUpClass(cls):
 @tagged('event_event')
 class TestEventData(TestEventInternalsCommon):
 
+    @users('user_eventmanager')
+    def test_event_type_configuration_from_type(self):
```

Comment:
To match naming used in this file, would rename to ``test_event_configuration_question_from_type`` (and move after test_event_configuration_note_from_type)